src/converse-bookmarks.js

```diff
@@ -253,7 +253,7 @@ converse.plugins.add('converse-bookmarks', {
             openBookmarkedRoom (bookmark) {
                 if ( _converse.muc_respect_autojoin && bookmark.get('autojoin')) {
                     const groupchat = _converse.api.rooms.create(bookmark.get('jid'), bookmark.get('nick'));
-                    if (!groupchat.get('hidden')) {
+                    if (!(groupchat.get('hidden') || groupchat.get('minimized'))) {
```

Comment:
For better readability, would you please change this from `!(x || y)` to `!x && !y`?

Reply:
done